controller.js

```diff
@@ -0,0 +1,13 @@
+const { selectTopics } = require(".//model");
+
+exports.getTopics = (req, res, next) => {
+  console.log("controller invoked");
```

Comment:
We no longer need these console logs, so make sure to remove them before pushing up to github :)